pywbem_mock/_instancewriteprovider.py

```diff
@@ -270,367 +209,280 @@ def __init__(self, cimrepository=None):
         Parameters:
 
           cimrepository (:class:`~pywbem_mock.BaseRepository` or subclass):
-            Defines the repository to be used by request responders.  The
-            repository is fully initialized.
+            The CIM repository to be used by mock WBEM server responders.
+            The repository must be fully initialized.
```

Comment:
Not sure what we mean by fully initialized.  Not sure that sentence adds anything.

Reply:
I agree. Removed.